src/__tests__/Routes/Details.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../../redux/store';
+import Details from '../../routes/Details';
+
+it('App component test', () => {
+  const tree = renderer
+    .create(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Details />
+        </Provider>
+      </BrowserRouter>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/67957533/227715045-c71f6b1f-4dec-4082-b43d-7eee5ee6e36c.png">

- [ ] Kindly fix this test case